spec/authentication.yml

```diff
@@ -106,6 +140,13 @@ components:
           schema:
             type: string
 
+    JWTToken:
```

Comment:
This response doesn't seem to be used anywhere. Is it meant to be referenced in responses of the new endpoint.